components/Heading/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export type Depth = 1 | 2 | 3 | 4 | 5 | 6;
+
+const CreateHeading = (depth: Depth) => {
+  const Tag: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' = `h${depth}`;
+
+  const Heading = ({ id, children, ...attrs }: React.PropsWithChildren<HTMLHeadingElement>) => {
+    if (!children) return '';
+
+    return (
+      // @ts-expect-error
+      <Tag {...attrs} className={`heading heading-${depth} header-scroll`}>
+        <div key={`heading-anchor-${id}`} className="heading-anchor anchor waypoint" id={id} />
+        <div key={`heading-text-${id}`} className="heading-text">
+          {children}
+        </div>
+        <a
+          key={`heading-anchor-icon-${id}`}
+          aria-label={`Skip link to ${children[1]}`}
```

Comment:
Not sure this `children[1]` logic still makes sense since [we seem to be getting rid of the `showAnchorIcons` option](https://github.com/readmeio/markdown/pull/883/files?file-filters%5B%5D=.jsx&file-filters%5B%5D=.tsx&show-deleted-files=true&show-viewed-files=true#diff-4cb87c52ddc885a869414f05bccd24cf6cfa74797a5ffdef9647ec0b286d7bb3L19-L20)? Maybe this should just be:

```jsx
aria-label={`Skip link to ${children}`}
```

Reply:
I didn't see it used anywhere, so I didn't bother porting it?